src/corelibs/U2View/src/ov_sequence/view_rendering/SequenceViewRenderer.cpp

```diff
@@ -59,7 +60,7 @@ int SequenceViewRenderer::posToXCoord(qint64 pos, const QSize&, const U2Region&
     CHECK(visibleRange.contains(pos) || pos == visibleRange.endPos(), -1);
 
     double res = (double)(pos - visibleRange.startPos) * getCurrentScale();
-    return qRound(res);
+    return qFloor(res);
```

Comment:
The same problem as in https://github.com/ugeneunipro/ugene/pull/1429, but for the backward function